### PR TITLE
Redesigning Arthropods 2.5: Add large dermatik nest, some dermatweaks

### PR DESCRIPTION
**Pull request by Venera3**
#### Summary

SUMMARY: Content "Add large dermatik nest, tweak dermatiks"

#### Purpose of change

Parasitoid wasps are terrifying, and dermatiks are one of the more interesting bugs in the game already with a fun hardcoded effect and whatnot. This PR gives them some more presence on the map and shows off their gruesome life cycle.

#### Describe the solution

- Adjusted dermatik larvae to grow up properly instead of using that ancient hardcoded special attack (24 day growth timer),
- Added the Mound of Dirt, a wilderness map special ( spawns 0-4 times per overmap ),
- Added some unfortunate hosts to the nests, along with dermatik midwives tending to them and defending the nest,
- Adjusted the rate of map extras in the swamps and removed some less sensible ones
- Did some tweaking on dermatiks themselves to be a bit larger and see better, but flee when hurt properly
 
#### Describe alternatives you've considered

Give them paralytic venom, but their real life counterparts use a *fascinating* symbiosis with certain viruses to incapacitate their unwilling cribs. I have some ideas how that could be simulated, but that's out of scope for this PR.
Have the mounds spawn ony in swamps, but I do have some other variants planned, and swamp specials tend to not be spawned consistently at the moment.
~~Use "spawns" instead of hand-placing monsters, but that field is either bugged or the least intuitive ever (issue to that effect follows).~~ Got it to do what I want, but still not 100% sure how.
Make the whole thing out of paper, but parasitoid wasps don't really do paper nests and it would have made it too easy to just burn them down. 3/4 of the interior walls are paper anyway, to communicate "wasps here" in a way the players are used to it.
Call them dermatik nests on the overmap, but the survivor correctly identifying stuff from miles ahead is a stretch already.
Tone down the body horror, but the game isn't called Happy Fun Days Ahead, and these things manage to creep *me* out.

#### Testing
Map spawns with the intended mobs, the dermatik incubators spawn larvae as babies and in "combat", larvae grow up, harvest gives the intended results, the interesting swamp map extras actually happen now.

#### Additional context
Derma1

The combat spawning from the hosts use a spell with a linked self-damage spell. As far as I know there's no way to supress the explosion effect from blast-type spells, and you can't define a message for the linked spell so far, so when they summon you get treated to a 3x3 explosion and get a nice thematic message followed by "the helpless dog shoots itself" for the self-damage spell. Figured that if the frog mother got away with that it was fine to use it here, but I'm open to ideas on how to make it nicer.

Swamp map extra changes:
- Normalized weigths to 1000 to keep the list readable
- Removed the clear cut, grove, shrubbery, and burned ground extras, since they didn't really work with the mapgen anyway,
- Lowered the chance of those weird desolate barns
- Gave a big boost to the pond's spawn rate, because the swamps were a bit too homogenous (this could stand to be even higher, but it works for now)
- Raised clay deposit chances to make them a consistent source
- Removed the college corpse spawn since it's, like,a bad place to hang, dude
- Raised the chances of normal, military and scientist corpses to show people fleeing the cities getting nommed / the military or scientists investigating the reports of Insects of Unusual Size pre-Cataclysm

This time I could refrain from changing any global swamp variables, so the knock-on effects for mods should be pretty minor. The removal of shrubbery, groves and burned grunds keeps them from diluting the extra pool, so mod stuff will show up a bit more.

I'm far from done with dermatiks, but my proper plans will require code work and this can stand on its own in the meantime. 
Part of #45030 .

### CODE REVIEW

File: data/json/furniture_and_terrain/terrain-walls.json [hunk 1]
```diff
@@ -903,6 +903,27 @@
       "items": [ { "item": "material_soil", "count": [ 100, 150 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_wall_dirt",
+    "name": "smooth dirt wall",
+    "looks_like": "t_wall_rammed_earth",
+    "description": "A solid wall of compressed dirt, mixed with…something to keep it stable.  Its surface glistens slightly, and it smells like rotten milk.",
```
Comment (Qrox) [inline data/json/furniture_and_terrain/terrain-walls.json:911]:
```suggestion
    "description": "A solid wall of compressed dirt, mixed with… something to keep it stable.  Its surface glistens slightly, and it smells like rotten milk.",
```

**Comment by actual-nh** [comment]
I am familiar with parasitoid wasps in general; which one/ones (re viruses and midwives) are these after?

**Comment by Venera3** [comment]
Thankfully (if you're anarthropod) there are no social species irl. However (eu)social lifestyles evolved at least three times independently in Hymenopterans, so I went with "generic parasitoid wasp that made the jump to communal nesting". The midwives are there to show a quick-and-dirty first step down the road of sister/worker sexual supression and with that to a true eusocial lifestyle, a bit nastier than the more sophisticated way of pheromones and/or beatings common in bees and wasps proper. Game design-wise they are there to have something more threatening than normal dermatiks in the nests.

**Comment by actual-nh** [comment]
I was just reading a bit on the _Polydnaviridae_, and will take a further look - very interesting! (As it happens, my doctorate is in Microbiology and Molecular Genetics, albeit with an emphasis on the latter.) Some of the wasps using them do appear to inject venoms also, incidentally.

Regarding paper and wasps - it's a pity the Orussidae (parasitoid wood wasps) are separate from the species using the viruses, at least as far as I can determine from a quick scan. While they don't produce wasp "paper", they would seem the most likely candidate among the parasitoid wasps. There's also the interesting case of wasps that are endoparasitic on paper wasps - [see here](https://blogs.biomedcentral.com/bugbitten/2018/11/16/remarkable-wasp-parasite) for a bit on those. (That's as well as "cuckoo" wasps, with their taking over of paper wasp nests by killing then imitating the dominant queen.)

**Comment by actual-nh** [comment]
> I was just reading a bit on the Polydnaviridae, and will take a further look - very interesting! (As it happens, my doctorate is in Microbiology and Molecular Genetics, albeit with an emphasis on the latter.) Some of the wasps using them do appear to inject venoms also, incidentally.

Heh. Regarding wasps vs zombies - the use of viruses may be a justification for their being able to parasitize zombies even if zombies are immune (or partially immune) to the venoms. (And people keep debating about whether viruses are living in the first place...)

> Regarding paper and wasps - it's a pity the Orussidae (parasitoid wood wasps) are separate from the species using the viruses, at least as far as I can determine from a quick scan. While they don't produce wasp "paper", they would seem the most likely candidate among the parasitoid wasps. There's also the interesting case of wasps that are endoparasitic on paper wasps - [see here](https://blogs.biomedcentral.com/bugbitten/2018/11/16/remarkable-wasp-parasite) for a bit on those. (That's as well as "cuckoo" wasps, with their taking over of paper wasp nests by killing then imitating the dominant queen.)

Oh! Is there some reason that dermatiks couldn't parasitoid other (paper-nest-building) giant wasps? This could involve anything from taking over an existing nest with only incubator usage of the other wasps (an intermediate stage on this could be interesting to encounter...) to inducing continuation of nest-building behavior in social wasps that weren't immediately used to incubate dermatik eggs...

**Comment by Venera3** [comment]
> Heh. Regarding wasps vs zombies - the use of viruses may be a justification for their being able to parasitize zombies even if zombies are immune (or partially immune) to the venoms. (And people keep debating about whether viruses are living in the first place...)

I did originally have zombies as incubators but decided against going that way because the speed of zombie evolution and their biological flexibility not really fitting the hapless victim role, plus seeing zombies suffer doesn't pack the same punch. If I manage to get the actual infection to work on other critters I intend to add a more mobile infested zombie to the tune of thorny shamblers/hollow zombies to show the Blob not really caring what happens to a zombie as long as they keep shuffling.

> 
> Oh! Is there some reason that dermatiks couldn't parasitoid other (paper-nest-building) giant wasps? This could involve anything from taking over an existing nest with only incubator usage of the other wasps (an intermediate stage on this could be interesting to encounter...) to inducing continuation of nest-building behavior in social wasps that weren't immediately used to incubate dermatik eggs...

As a matter of fact, dermatiks are currently in the evolution line of the small wasps, which is either a pretty lazy shortcut or a stroke of genius to show parasitoidism. When wasps get their eggs and proper life cycle they could very well have a chance to hatch into dermatiks (maybe from a variant queen?) to show an alternative brood parasitic way of dermatik reproduction. We'll see how it plays out.

**Comment by Venera3** [comment]
Okay, I poked at `"spawns":` until I got some nice distribution around the nests as well. I still don't have any idea how that value works, but now you have 2-3 dermatiks hangin' around the perimeter ( 3-4 OMTs) of the mound. Also raised the city distance value.

**Pull request merged**